server/services/location.js

```diff
@@ -8,12 +8,17 @@ async function find (location) {
   const url = util.format(bingUrl, query, bingKey)
   let data = await getJson(url, true)
 
-  if (data === undefined) {
+  if (!data) {
     throw new Error('Invalid data returned from third party location search')
   }
 
+  // Check for OK status returned
```

Comment:
Bing returns a statusCode with the result which we don't check! So we need some error handling around non 200 responses